### PR TITLE
Format team awakening totals

### PR DESCRIPTION
**Pull request by Siveran**
- Formats team awakenings into a grid
- Adds more awakenings (rows, unbind, etc.)
- Shows missing awakenings
- Removes the nested subtables in the team stats table
- Aligns stats with the right edges of the card's icon 
- Refactors team stats table generation
- Adds padding to totals to match the old Valeria

Minor adjustments, all around!

### CODE REVIEW

File: src/templates.ts [hunk 1]
```diff
@@ -1672,37 +1673,58 @@ class TeamPane {
...
-      const aggregatedAwakeningCount = create('span') as HTMLSpanElement;
-      aggregatedAwakeningCount.innerText = 'x0';
-      this.aggregatedAwakeningCounts.set(awakening, aggregatedAwakeningCount);
-      container.appendChild(aggregatedAwakeningCount);
-      this.statsEl.appendChild(container);
+    const awakeningTable = create('table', ClassNames.AWAKENING_TABLE) as HTMLTableElement;
+    for (const awakeningSet of awakeningsToDisplay) {
+      const awakeningRow = create('tr') as HTMLTableRowElement;
+      for (const awakening of awakeningSet) {
+        const container = create('td') as HTMLSpanElement;
```
Comment (mtkwock) [inline src/templates.ts]: You're making an HTMLTableCellElement with td.  Although It doesn't vastly change the outcome, please fix.

File: src/templates.ts [hunk 2]
```diff
@@ -1672,37 +1673,58 @@ class TeamPane {
...
+        const awakeningIcon = create('span', ClassNames.AWAKENING) as HTMLSpanElement;
+        const [x, y] = getAwakeningOffsets(awakening);
+        awakeningIcon.style.backgroundPosition = `${AwakeningEditor.SCALE * x}px ${AwakeningEditor.SCALE * y}px`;
+
+        container.appendChild(awakeningIcon);
+        const aggregatedAwakeningCount = create('span') as HTMLTableCellElement;
```
Comment (mtkwock) [inline src/templates.ts]: This is an HTMLSpanElement

**Review by mtkwock — COMMENTED** [review]
Overall good, just one smaller fix for the sake of typing.

**Comment by Siveran** [comment]
Good catches on the typing! I've fixed both of those.

**Review by mtkwock — APPROVED** [review]

**Pull request merged**